.github/workflows/audit.yml

```diff
@@ -0,0 +1,28 @@
+name: Security audit
+on:
+  pull_request:
+    paths: Cargo.lock
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
it runs both on each PR (in case a new vulnerability is discovered) and nightly at 00:00 (in case a new vulnerability is discovered)